modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/Checkpointer.java

```diff
@@ -898,4 +894,16 @@ CompletableFuture<Void> prepareToDestroyPartition(GroupPartitionId groupPartitio
 
         return processedPartitionFuture == null ? nullCompletedFuture() : processedPartitionFuture;
     }
+
+    private void syncWal(CheckpointMetricsTracker tracker) throws IgniteInternalCheckedException {
+        try {
+            tracker.onSyncWalStart();
```

Comment:
In Ignite 2, the log was a WAL and this term was used in the whole codebase. In Ignite 3, we usually talk about Raft log (or simply log) and not a WAL. Maybe it will be more consistent to use 'log' term instead of 'WAL' in this class as well.